debian/base/Dockerfile.rpm

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
It's a bit weird to have centos Dockerfile inside of the "debian" folder. 

Can we do either of the following? 
1. Create a new "centos" folder so we have centos/base/Dockerfile and then toggle everything based on "centos" vs "debian" (instead of "" vs "rpm"). Alternatively do "rpm/base" vs "deb/base". 
2. Or  reverse the grouping so we get "base/rpm/Dockerfile" vs "base/deb/Dockerfile"